asyncsleepiq/fuzion/foundation.py

```diff
@@ -49,13 +49,13 @@ async def init_features(self) -> None:
             await self.init_lights()
         if self.features["articulationEnableFlag"]:
             await self.init_actuators()
-        await self.init_presets()
+        await self.init_presets({})

```

Comment:
This change exists in a few places - while `data` isn't needed for the fuzion versions of these functions, the parameter is still required, and you'll get errors if you don't provide it.

Not sure the best solution here, as maintaining parity between the two classes is hard when one requires the data upfront, and the other makes the network request for it.